src/index.html

```diff
@@ -10,8 +10,179 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit"
+      >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      onsubmit="onSubmit"
    >
```
formatting (checklist point 3)
+ 
![image](https://user-images.githubusercontent.com/64437769/142170578-511c1f41-67d4-479a-96ab-50a4b28a983f.png)